README.md

```diff
@@ -146,3 +151,15 @@ The glow-monitor assumes that there will be at least 30 minutes of network
 uptime every day during the daylight, with at most 2 consecutive days of
 downtime. More downtime than that will result in missing power reports that
 under-represent a solar farms contributions.
+
+## File Writing and Archiving
+
+To ensure consistency of files stored on the server, they should be written
+append-only, using a single write call. This allows consistent read access
+without the need for file locking.
+
+The archive strategy is to return all public data as files, providing
+them in a zip archive. In case of updates during the archive
+process, files are archived in the following order to maintain
+consistency: server public keys, device stats, equipment authorization,
+equipment reports.
```

Comment:
I believe this order is incorrect. You want to archive the equipment reports before you archive the equipment authorizations, otherwise it's possible to get an equipment report in the archive that references an authorization that didn't exist when the snapshot of the authorizations was taken.

Reply:
Right. Change order would be public keys, equipment auth, equipment reports, then device stats. So the archive should be in reverse order to this

Reply:
This conversation was marked as resolved but the README still has the incorrect order for the reports.

Reply:
This comment still isn't fixed

Reply:
Thanks I missed it